tests/integration/tier0/non-destructive/duplicate-pkgs/main.fmf

```diff
@@ -0,0 +1,19 @@
+summary+: |
+    Duplicate pkgs installed on the system
+description+: |
+    Verify that the conversion does not crash when the same
+    package (of different version) is installed on the system.
+    Verify that the proper inhibitor is raised.
+
+
+/duplicate-pkgs:
+    summary+: |
+        Verify the https://issues.redhat.com/browse/RHELC-1070. The bug is reproducible only
+        on EL7 distros. On EL8 the issue is handled by the DNF itself.
+    adjust+:
+        - enabled: false
+          when: distro == centos-8, alma-8, rocky-8, oracle-8
```

Comment:
```suggestion
          when: distro == centos-8, alma-8, rocky-8, oracle-8
```
Please add the `because:` section.
(e.g. "Reproducible on/reported for 7.9 RHEL7-like distributions only". Or what you have in the summary The bug is reproducible only on EL7 distros. On EL8 the issue is handled by the DNF itself.)

Reply:
Right, I added the `because:` section